getting-started/proc-add-source-control-rhdh-catalog.adoc

```diff
@@ -284,11 +284,12 @@ data:
         githubOrg:
           default:
             id: production
-            orgUrl: '${GITHUB_ORG_URL}'
+            githubUrl: "https://github.com"
+            orgs: [ "${GITHUB_ORG}" ]
```

Comment:
```suggestion
            orgs: [ "${GITHUB_ORG}" ] # <1>
```
For more information please see: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks